backend/models/dynamodb/Entity.js

```diff
@@ -0,0 +1,87 @@
+
+import DynamoDB from "@klaudsol/commons/lib/DynamoDB"
+import { 
+  DYNAMO_DB_TYPES, 
+  DYNAMO_DB_INDEXES, 
+  DYNAMO_DB_TABLE, 
+  DYNAMO_DB_ORDER 
+} from "@/constants";
+
+export default class Entity {
+  // Gets all items where entity = entity_type_slug (eg. tech)
+  // and if type = content
+  // Sample: api/tech
+  // TODO: Add pagination
+  // TODO: Add filtering
+  // TODO: Add search
+  static async whereEntityTypeSlug({ entity_type_slug, order }) {
+    const db = new DynamoDB();
+
+    let scanIndexForward = true;
+    if (order === DYNAMO_DB_ORDER.desc) {
+      scanIndexForward = false;
+    } 
+
+    const params = {
+      TableName: DYNAMO_DB_TABLE,
+      IndexName: DYNAMO_DB_INDEXES.entity_type_order,
+      KeyConditionExpression: "#entity_type = :entity_type",
+      FilterExpression: "#type = :type",
+      ExpressionAttributeNames: {
+        "#entity_type": "entity_type",
+        "#type" : "type",
+      },
+      ExpressionAttributeValues: {
+        ":entity_type": { S: entity_type_slug },
+        ":type": { S: DYNAMO_DB_TYPES.content },
+      },
+      ScanIndexForward: scanIndexForward
+    };
+    const result = await db.query(params);
+    return result;
+  }
+
+  // Finds an item based on its entity_type_slug and slug
+  // Sample: api/tech/aws
+  static async findByEntityTypeSlugAndSlug({ entity_type_slug, slug }) {
```

Comment:
I realized that putting the slug in the `entity` field is not a good idea. This would be evident if the user tries to change the slug of an entity type - we will need to update all the entities of that type 😱.

What I suggest is to do 2 lookups:

1.) Find the PK of the content_type where slug = :entity_type_slug.
2.) Find the content where PK = content_type#id (taken from 1) and slug = :slug.

Performance-wise, I think this is okay, as even if we have 1 million records, we still invoke exactly 2 queries.